themes/default_symbols.fish

```diff
@@ -33,6 +33,9 @@ set FLSYM_ROOT_USER         "\u2192"
 # Symbol for VFISH segment
 set FLSYM_VFISH             "\u2635"
 
+# Symbol for CONDA segment
+set FLSYM_CONDA             "\U0001F40D"
```

Comment:
I am not sure about using emojis directly in the default theme, some of our userbase is running on Linux with a poor support for emojis in the terminal. If you can find a more fitting icon this would be perfect !

*Note*: I am not against it per say, its just for a compatibility issue, in the end anybody would be able to overwrite it in their `config.fish` to use the emoji version ;)